statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2894,6 +2894,76 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def get_rsquared(self, baseline="rwdrift"):
+        if self.model.k_endog > 1:
+            raise NotImplementedError(
+                "To be Implemented for Multivariate Model")
+        if baseline == "rwdrift":
+            return self.rsquared_rwdrift
+        elif baseline == "mean":
+            return self.rsquared_mean
+        elif baseline == "seasonal":
+            raise self.rsquared_seasonal
+        else:
+            raise NotImplementedError(
+                "baseline must be in ['rwdrife', 'mean', 'seasonal']")
+
+    @cache_readonly
+    def sse_finite(self):
+        d = np.maximum(self.loglikelihood_burn, self.nobs_diffuse)
+        srss = np.sum(self.standardized_forecasts_error[0, d:]**2)
+        f_T = self.forecasts_error_cov[0, 0, -1]
+        return f_T * srss
+
+    @cache_readonly
+    def rsquared(self):
+        """
+        In state space models there is not a single comparison
+        model that will always be useful for rsquared computation.
+        However, we have implmented `rsquared_rwdrift` and `rsquared_mean`
+        It is recommended to use `rsquared_rwdrift`
+        """
+        raise NotImplementedError(
```

Comment:
I think that the text you have as the docstring should be the message given in the `NotImplementedError`